mgmt/api/CoreAPIRemote.cc

```diff
@@ -1022,6 +1022,21 @@ EventSignalCbUnregister(const char *event_name, TSEventSignalFunc func)
   return TS_ERR_OKAY;
 }
 
+TSMgmtError
+HostStatusSet(const char *host_name, bool value)
```

Comment:
Rather than a `bool`, it's more obvious to just send the enum value (as a `MgmtMarshalInt`).